helix-core/src/test/java/org/apache/helix/integration/rebalancer/TestMixedModeAutoRebalance.java

```diff
@@ -211,7 +211,6 @@ public void testUserDefinedPreferenceListsInFullAutoWithErrors() throws Exceptio
       }
       return false;
     }, 2000);
-    Assert.assertTrue(_clusterVerifier.verify(3000));
```

Comment:
Because we adjust the behavior of strict ev verifier again in this PR. Basically, change it back to verify based on the preference list. So it will never converge to mapping with ERROR state.

Note that is changing the behavior back to fit the master branch code.